.github/workflows/build-and-deploy-preprod.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Kjør ktlint
         env:
           GITHUB_USERNAME: x-access-token
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.READER_TOKEN }}
```

Comment:
READER_TOKEN fordi baks-dogen er private?

Reply:
For hvis det er det, så tror jeg vi fint kan åpne repoet, eller så må samme endring til i prod-løypa

Reply:
Stemmer. La til samme endring i prod-løypa 👍